source/docs/12-facsimilesrecordings.xml

```diff
@@ -299,6 +299,14 @@
                      </figure>
                   </p>
                   <p>indicates that the entities identified in <att>data</att> all occur at the same instant.</p>
+                  <p><att>extData</att> is a container for holding non-MEI data formats, similar to <att>extMeta</att> but available in <att>when</att> rather than in MEI.header. <att>extData</att> allows for data from audio or other sources to be linked to notes or other score events. Data should be enclosed in a CDATA tag.</p>
```

Comment:
```suggestion
                  <p><gi>extData</gi> is a container for holding non-MEI data formats, similar to <gi>extMeta</gi> but available in <gi>when</gi> rather than in <gi>meiHeader</gi>. <gi>extData</gi> allows for data from audio or other sources to be linked to notes or other score events. Data should be enclosed in a CDATA tag.</p>
```

Reply:
Replace the last sentence with "When the data contains the less-than or ampersand characters or significant whitespace, such as line breaks, then it should be enclosed in a CDATA section."